src/icons/observations-disabled.svg

```diff
@@ -0,0 +1,26 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="22.185" height="26" viewBox="0 0 22.185 26">
+  <defs>
+    <style>
+      .cls-1 {
+        fill: none;
+        stroke: #cde0c9;
```

Comment:
This is what you can change through props with styled components if you use it as `svg` you'd then only have one svg asset.